COLLECTOR_VERSION

```diff
@@ -1 +1 @@
-3.14.1
+3.14.x-56-gb9058f4fad
```

Comment:
There is 3.14.x-77 already on
- https://github.com/stackrox/stackrox/pull/5488

Reply:
We would still need to bump it at least once (there are some more fixes are coming to the Falco fork), so if you don't mind I'll merge this one now and take care of the consequent version updates.